mythril/laser/smt/bitvec.py

```diff
@@ -242,8 +641,16 @@ def Concat(*args: Union[BitVec, List[BitVec]]) -> BitVec:
 
     nraw = z3.Concat([a.raw for a in bvs])
     annotations = [] # type: Annotations
+    bitvecfunc = False
     for bv in bvs:
         annotations += bv.annotations
+        if isinstance(bv, BitVecFunc):
+            bitvecfunc = True
+
+    if bitvecfunc:
+        # Is there a better value to set name and input to in this case?
+        return BitVecFunc(raw=nraw, name=None, input=None, annotations=annotations)
```

Comment:
if name=None is possible, in that case type of name should be Optional[str](in the \_\_init\_\_() )